round4/group16

```diff
@@ -0,0 +1,60 @@
+Chrome Extension Idea: AI Research Assistant
+Authors
+Runzhi Gu, Sam Bhatta, Daniel Kwon, Caroline Begg
+
+Problem Statement
+Students, researchers, and information-seekers often struggle with the overwhelming task of sifting through vast amounts of online content to find relevant information. There is a lack of effective tools that are embedded in the web browser that helps the users quickly understand a long and complicated web article, which makes the research process daunting and less productive.
+
+Target Audience
+Students conducting research for academic papers or projects. Researchers gathering up to date information across various fields. Educators and content creators that collect rephrased information. Anyone who frequently engages in extensive online reading and needs to compile information efficiently. 
+
+Description
+The AI Research Assistant is a Chrome Extension powered by Open AI api through the GPT model, designed to enhance the online research process. It offers a sidebar interface that allows users to summarize web pages or specific sections, and displays it in an organized and refined manner. 
```

Comment:
I like this idea a lot! It almost sounds too good to be true... Does something identical already exist?

Otherwise, I'd recommend looking at chrome extensions like [Merlin](https://chromewebstore.google.com/detail/merlin-1-click-access-to/camppjleccjaphfdbohjdohecfnoikec) for inspiration.